src/components/dialogs/nuxs/NeueTypography.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import {View} from 'react-native'
+import {msg, Trans} from '@lingui/macro'
+import {useLingui} from '@lingui/react'
+
+import {AppearanceToggleButtonGroup} from '#/screens/Settings/AppearanceSettings'
+import {atoms as a, useAlf, useTheme} from '#/alf'
+import * as Dialog from '#/components/Dialog'
+import {useNuxDialogContext} from '#/components/dialogs/nuxs'
+import {Divider} from '#/components/Divider'
+import {TextSize_Stroke2_Corner0_Rounded as TextSize} from '#/components/icons/TextSize'
+import {TitleCase_Stroke2_Corner0_Rounded as Aa} from '#/components/icons/TitleCase'
+import {Text} from '#/components/Typography'
+
+export function NeueTypography() {
+  const t = useTheme()
+  const {_} = useLingui()
+  const nuxDialogs = useNuxDialogContext()
+  const control = Dialog.useDialogControl()
+  const {fonts} = useAlf()
+
+  Dialog.useAutoOpen(control, 3e3)
```

Comment:
Curiosity. What is the purpose of the `setTimeout`? I do recall having to do this for intents, but it was a lot shorter than three seconds if I remember correctly. Mainly:

- Why did we pick three seconds?
- Is there cases where this wouldn't be long enough?
- Is it possible the user could start scrolling the app before this gets presented to them, and the dialog would sort of get in the way?

Reply:
Opening right away is fine, I just liked the delay. Naturally any delay could cause it to pop up after scrolling. How do you feel about it?

Reply:
Lets see how it feels on device.